packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -203,14 +205,22 @@ exports.sourceNodes = async (
   // Default skipFileDownloads to false.
   skipFileDownloads = skipFileDownloads || false
 
+  // Default language to `en`
+  language = language || `en`
+
+  // Default defaultLanguage to `en`
+  defaultLanguage = defaultLanguage || `en`
+
+  const languagePrefix = defaultLanguage !== language ? language : ``
+
   // Fetch articles.
   reporter.info(`Starting to fetch all data from Drupal`)
 
   drupalFetchActivity.start()
 
   let allData
   try {
-    const data = await axios.get(`${baseUrl}/${apiBase}`, {
+    const data = await axios.get(`${baseUrl}/${languagePrefix}/${apiBase}`, {
```

Comment:
Will this always work? It seems that this will default to "/en/jsonapi" which doesn't seem to work out of the box with JSON:API.

I think the better approach might be to default to the current URL (baseURL/apiBase) unless a language prefix is added to gatsby-config.js. If it's added then it can be appended to the URL.

Reply:
I saw the same thing, fortunately Drupal does redirect /// to / so in the case a languagePrefix is not specified it would change to `baseUrl//jsonapi` rather than `baseUrl/es/jsonapi` which both work. I don't like forcing a redirect in default case though so I will definitely refactor this so that it's not required when langaugePrefix is not specified.